### ISSUE TITLE
libraw_dcraw_make_mem_image takes 2 arguments, one given

### ISSUE THREAD

**Issue by jim-easterbrook** [photoshell/rawkit#91]
My first attempt to use rawkit failed immediately (I'm good at breaking things). Fresh install with pip, then tried to use raw.to_buffer():
```
...
    data = raw.to_buffer()
  File "/usr/lib/python2.7/site-packages/rawkit/raw.py", line 139, in to_buffer
    processed_image = self.libraw.libraw_dcraw_make_mem_image(self.data)
TypeError: this function takes at least 2 arguments (1 given)
```

**Comment by SamWhited** [comment]
Oops, we're not handling the error code properly (this method, and the similar method for thumbnails, take a second argument, a pointer which will contain the error code).

Thanks for the report!

@campaul ^^

**Comment by campaul** [comment]
The fix for this is already in master. @SamWhited can you cut a new release so people can get a working version from pip?

**Comment by SamWhited** [comment]
Oops, yah, fixed in cc7ab73fd7d0bc838f25d93ad4f60b182e08116b

Will cut a new release tonight.

**Comment by jim-easterbrook** [comment]
Thanks - I probably should have tried installing from source before raising this.

**Comment by campaul** [comment]
I'm going to fix #88 (which relates to this) before we cut another release, but I'll make sure to get that done tonight.

**Comment by SamWhited** [comment]
Just cut version 0.4.2, which contains the fix for this problem as well as the fix for #88 (the error code is actually handled).

**Comment by jim-easterbrook** [comment]
Thanks - appears to be working perfectly for me now.
